packages/grafana-ui/src/components/Drawer/Drawer.tsx

```diff
@@ -146,6 +150,36 @@ export function Drawer({
   );
 }
 
+function useResizebleDrawer(): [number | undefined, React.EventHandler<React.MouseEvent>] {
+  const [drawerWidth, setDrawerWidth] = useState<number | undefined>(undefined);
+
+  const onMouseMove = useCallback((e: MouseEvent) => {
+    let offsetRight = document.body.offsetWidth - (e.clientX - document.body.offsetLeft);
+    let minWidth = 256;
+    if (offsetRight > minWidth) {
```

Comment:
@ashharrison90 made some refinements so the width calc when dragging ends up being a width percentage (in vw units), that way the size stays relative when you resize your browser (and never goes beyond 100% of viewport). Also added Math.min so that that width never goes beyond 98%